test/parallel/test-cluster-worker-disconnect.js

```diff
@@ -40,7 +40,8 @@ if (cluster.isWorker) {
 
   // Disconnect worker when it is ready
   worker.once('listening', common.mustCall(() => {
-    worker.disconnect();
+    const w = worker.disconnect();
+    assert.strictEqual(worker, w, 'did not return a reference');
```

Comment:
No worries :) Do you think the assertion is only needed in one test file instead of two?